src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.test.ts

```diff
@@ -0,0 +1,217 @@
+import languageAssertions from '../../../../../test/unit/utils/languageAssertions';
+import { FormContent, FormFields, FormOptions, LanguageLookup } from '../../../../app/form/Form';
```

Comment:
Please move this file under screening-questions. so the folder structure will be c100-rebuild/screening-questions